services/video_chess_indexer/main/main.py

```diff
@@ -1,5 +1,3 @@
-from yolov5.detect import main
-
-main()
-
+from yolov5.detect import main, run
```

Comment:
Don't need to import main here anymore, right?

Reply:
Yep - Took it out there.